examples/structured_2d_dgsem/elixir_advection_coupled.jl

```diff
@@ -31,6 +31,9 @@ using Trixi
 #     └────────────────────┴────────────────────┘
 # (-1, -1)                                   ( 1, -1)
 
+# Define identical resolution such that it is easier to change from `trixi_include`
+cells_per_dimension = (8, 16)
```

Comment:
Could we move this closer to the mesh so that it doesn't split the mesh definition too much?

Reply:
Fixed in 15b92deab